buildpacks/nodejs-corepack/src/main.rs

```diff
@@ -63,6 +63,7 @@ impl Buildpack for CorepackBuildpack {
         })
     }
 
+    #[allow(deprecated)]
```

Comment:
Is this addition intentional, or a rebase issue?

```suggestion
```

Reply:
This `#[allow(deprecated)]` annotation is now removed (again).